IVehicle.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeReviewDemo
+{
+    internal interface IVehicle
+    {
+        void StartEngine();
+        void StopEngine();
+        void Accelerate();
+        void Brake();
+
```

Comment:
The `PrepareCoffee` and `AddCoffee` methods do not seem relevant to a vehicle interface. Consider removing these methods from the `IVehicle` interface.